### PR TITLE
Update developer guide

### PR DESCRIPTION
**Pull request by mattcollier**
Just want to correct gross inaccuracies before Bee announcement on Monday.

It is my assumption that did-whisper probably needs some TLC before working with Bee, so I have removed the link to it for now.

### CODE REVIEW

**Review by dmitrizagidulin — APPROVED** [review]
👍 thanks @mattcollier!
(Also, I didn't realize Bee announcement is going out Monday, whoops.)

**Review by gannan08 — APPROVED** [review]

File: docs/developers/guides/create-did.md [hunk 1]
```diff
@@ -5,16 +5,23 @@ identifier using command line tools.
 
 ## Requirements
 
-* [Node v8.6+](https://nodejs.org/en/download/)
+* [Node.js v8.6+ and Node.js v10.x](https://nodejs.org/en/download/).
```
Comment (davidlehn) [inline docs/developers/guides/create-did.md]:
```suggestion
* [Node.js v8.6+ or Node.js v10.x](https://nodejs.org/en/download/).
```

File: docs/developers/guides/create-did.md [hunk 2]
```diff
@@ -5,16 +5,23 @@ identifier using command line tools.
...
 ## Requirements
 
-* [Node v8.6+](https://nodejs.org/en/download/)
+* [Node.js v8.6+ and Node.js v10.x](https://nodejs.org/en/download/).  
 
-## Install did-client
+### Coming Soon
 
-Install the Decentralized Identifier client by running the following command:
+Node v12 support will be added in the upcoming v0.5 release.
```
Comment (davidlehn) [inline docs/developers/guides/create-did.md]:
```suggestion
Node.js v12 support will be added in the upcoming v0.5 release.
```

File: docs/developers/guides/create-did.md [hunk 3]
```diff
@@ -5,16 +5,23 @@ identifier using command line tools.
...
+
+## Install did-cli
+did-cli is a command line tool that allows the user to interact with the
```
Comment (davidlehn) [inline docs/developers/guides/create-did.md]:
```suggestion
`did-cli` is a command line tool that allows the user to interact with the
```

**Review by davidlehn — APPROVED** [review]

**Pull request merged**